ci-operator/config/openshift-priv/multus-networkpolicy/openshift-priv-multus-networkpolicy-cni-release-4.8.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
```

Comment:
PLease remove all files and config for `openshift-priv/multus-networkpolicy` and keep just the ones for `openshift/multus`. The config for `openshift-priv` is completely inferred from the one for the public repo (see https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization for more information)

Reply:
Petr, thanks for the eyes here! I wasn't aware of that. appreciate, got those removed.